lib/src/models/subreddit_moderation.dart

```diff
@@ -346,17 +346,19 @@ class SubredditSettings {
   String get submitText => _data['submit_text'];
   set submitText(String x) => _data['submit_text'] = x;
 
-  // TODO(bkonyi): figure out what this is.
-  // get spamLinks => _data['spam_links'];
+  // Returns the filter strength set for spam links for subreddit.
+  // Possible values: all, low, high.
```

Comment:
Couple of things here:

If there's only a few possible values here, we'll probably want to define an enum and return that (this is what we do throughout most of the package). Not sure what the best name would be here though... maybe `SpamSensitivity`?

Second, any documentation should be prefaced with a `///` for `dartdoc` to pick it up when generating the API documentation.

Reply:
For the first part, I did initially had a TODO comment for the Enum, but dropped it because i wasnt sure. Also `SpamSensitivity` is a pretty good name and should work for the multiple similar fields.

Reply:
For the second part, i didnt put `///` because the other fields arent documented. Guess i will document the fields whose usage cant be discerned just by the name.